### DISCUSSION TITLE
Styled-Components

### DISCUSSION THREAD

**Discussion by jlavrin** [vercel/turborepo#2845] [Help] [answered]
### Is there any possibility to use Styled-Components with Turbopack?
 I tried to start app **without** 
`
compiler: {
    styledComponents: true
}
`
setting in next.config.js so component's classes rendered at server and client doesn't match.

![image](https://user-images.githubusercontent.com/83779054/204146371-68c229f3-8bbf-4187-a8c0-d5e961231ea9.png)

So I tried to start app with compiler setting `styledcomponents: true` and now Turbopack is not happy at all as it throws:

_'Error: You are using configuration and/or tools that are not yet
supported by Next.js v13 with Turbopack:'_

Is there any way to solve this and use SC with Turbo?

**Comment by anthonyshew** [comment, marked as answer]
If you are still seeing this error, please create an issue!
